solr/conf/schema.xml

```diff
@@ -315,7 +314,6 @@
     <fieldType class="solr.TextField" name="textSuggest" positionIncrementGap="100">
       <analyzer>
         <tokenizer class="solr.KeywordTokenizerFactory"/>
-        <filter class="solr.StandardFilterFactory"/>
```

Comment:
This is getting taken out in Solr 8. I was trying my hand at getting that across the line last night/this morning but it wasn't working so I cut my losses. I did however forget to add this back in and can if you want.

Reply:
I think you have to if you want search to work. it won't tokenize properly in 7 otherwise